docs/config.md

```diff
@@ -809,7 +809,7 @@ which lets you use [AWS Route 53](https://aws.amazon.com/route53/) as the challe
 HTTP challenge. I've found [this guide](https://nandovieira.com/using-lets-encrypt-in-development-with-nginx-and-aws-route53) to
 be incredibly helpful.
 
-### nginx/Apache2/caddy
+### nginx/Apache2/caddy/ferron
```

Comment:
This link sadly cannot be changed because it is baked into the Android app, afaik. I can accept the PR but without this title change.